apply_recommendations/cmd/apply/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	"github.com/hashicorp/terraform-exec/tfexec"
+)
+
+func main() {
+	workingDir := flag.String("working-dir", "./", "path to working directory")
+	dryRun := flag.Bool("dry-run", false, "dry run; run terraform plan instead of terraform apply")
```

Comment:
personally, i would default this to `true`, but that might be personal preference.